src/qt/coincontroldialog.cpp

```diff
@@ -379,11 +378,11 @@ void CoinControlDialog::viewItemChanged(QTreeWidgetItem* item, int column)
         COutPoint outpt(uint256S(item->text(COLUMN_TXHASH).toStdString()), item->text(COLUMN_VOUT_INDEX).toUInt());
 
         if (item->checkState(COLUMN_CHECKBOX) == Qt::Unchecked)
-            coinControl->UnSelect(outpt);
+            CoinControlDialog::coinControl()->UnSelect(outpt);
```

Comment:
No need to prefix with  `CoinControlDialog::` here, this is still in the class.

Reply:
I know, but this way it's clear it's a static member.

Reply:
Let's not do that, we intend to make it non-static field later anyway, so this seems moving the wrong direction.